conans/client/cmd/export.py

```diff
@@ -47,8 +48,10 @@ def cmd_export(app, conanfile_path, name, version, user, channel, graph_lock=Non
     mkdir(export_folder)
     mkdir(export_src_folder)
     origin_folder = os.path.dirname(conanfile_path)
-    export_recipe(conanfile, origin_folder, export_folder)
-    export_source(conanfile, origin_folder, export_src_folder)
+    setattr(conanfile, "conanfile_folder", origin_folder)
```

Comment:
Why a ``setattr``? Reads weird, why not ``conanfile.conanfile_folder = origin_folder``?

Also, why not using the already existing ``recipe_folder``?